### PR TITLE
To Drink or Not to Drink

### PR DESCRIPTION
**Pull request by razvanges**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2182)

### CODE REVIEW

**Comment by razvanges** [comment]
Please review this, @casey-collab

File: romanian/nalba.md [hunk 1]
```diff
@@ -37,4 +37,23 @@ Se pusera ei pe baut ceai de nalba cand deodata...
...
+pantofi albastrii...
+
+Bun, bun, buuun! zise zana. Stiam eu ca ma pricep la "d-astea" :-D
+
+Ihi, mormai Ginel, cu gura cascata si cu ochii fixati undeva, adanc.
+
+Haide, ia si tu o gura, ca imi esti simpatic.
+
+Pai, se puse pe ganduri Ginel, [sa beau](potiune_fermecata/beau/beau.md) sau... 
+[sa nu beau](potiune_fermecata/nu_beau/nu_beau.md)?
```
Comment (casey-collab) [inline romanian/nalba.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by razvanges** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :crown:

**Pull request merged**